bip-tap.mediawiki

```diff
@@ -521,34 +523,39 @@ An asset leaf is a serialized TLV blob, with the following key-value mappings:
 * type: 9 (<code>asset_script_key</code>)
 ** value: 
 *** [<code>33*byte</code>:<code>pub_key</code>]
-* type: 10 (<code>asset_group_key</code>)
-** value: 
+* type: 10 (<code>asset_group_key</code>) [version = 0, deprecated]
```

Comment:
Maybe better to spell out the V1 Asset TLV explicitly, since the type numbers should also be updated to follow the even/odd rule?

Reply:
I think we'd need to first step back and agree on what even/odd in this context means. Is it something like:
  * odd is for things that just need to be serialized properly in order to validate sigs 
  * even is for things that need to be known in order to properly validate a state transition or other higher level protocol rules